sections/03_categorize.md

```diff
@@ -231,6 +194,44 @@ for such evaluations. We recommend the use of an independent test set wherever
 it is feasible. Despite this minor limitation, the work clearly illustrates the
 potential that can be unlocked from images stored in electronic health records.
 
+Chest X-rays are a common radiological examination for screening and diagnosis
+of lung diseases. Although hospitals have accumulated a large number of raw
+radiology images and reports in Picture Archiving and Communication Systems and
+their related reports in Radiology Information System, it is not yet known how
+to effectively use them to learn the correlation between pathology categories
+and X-rays. In the last few years, deep learning methods showed remarkable
+results in chest X-ray image analysis [@arxiv:1701.06599 @arxiv:1705.02315].
+However, it is both costly and time-consuming to annotate a large-scale fully-
+labeled corpus to facilitate the data-hungry deep learning models. As an
+alternative, Wang et al. [@arxiv:1705.02315] proposed to use weakly labeled
+images for training deep learning models. To generate weak labels for X-ray
+images, they applied a series of Natural Language Processing (NLP) techniques to
+the associated chest X-ray radiological reports. Specifically, they first
+extracted all diseases mentioned in the reports using a state-of-the-art NLP
+tool, then applied a newly developed negation and uncertainty detection tool
+(NegBio) to filter negative and equivocal findings in the reports. Evaluation on
+three independent datasets demonstrated that NegBio is highly accurate for
+detecting negative and equivocal findings (~90% in F-measure overall). These
+highly accurate results meet the need to generate a corpus with weak labels,
+which serves as a solid foundation for the later process of image
+classification. The resulting dataset (CXR-XIV
+[@url:https://console.cloud.google.com/storage/browser/gcs-public-data--nih/radiology_2017/Chest_X-Ray_CVPR17])
+consists of 108,948 frontal-view chest X-ray images (from 32,717 patients) and
+each image is associated with one or more weakly-labeled pathology category
+(e.g. pneumonia and cardiomegaly) or "normal" otherwise. Further, Wang et al.
+[@arxiv:1705.02315] used this dataset with a unified weakly-supervised
+multi-label image classification framework, to detect common thoracic diseases.
+It showed superior performance over a benchmark using fully labeled data.
+
+We illustrated augmentation with adversarial examples in the mammography domain,
+but Geras et al. [@tag:Geras2017_multiview_mamm] showed that both using a large
```

Comment:
They have a larger training dataset than most of the studies we highlight but still use data augmentation. How much do you think that this paragraph adds? I am tempted to drop it, since we are way over for length and it largely hits the same ideas that we've covered above..